zipline/__main__.py

```diff
@@ -317,13 +320,6 @@ def run(ctx,
         benchmark_spec=benchmark_spec,
     )
 
-    if output == '-':
```

Comment:
why can we remove this logic?

Reply:
The final lines of `_run` (called above) are:
```python
    if output == '-':
        click.echo(str(perf))
    elif output != os.devnull:  # make the zipline magic not write any data
        perf.to_pickle(output)

    return perf
```